src/components/panel/accountSubComponents/AccountLinkToUi.vue

```diff
@@ -61,138 +61,138 @@
-      },
-      specificUiUrl() {
-        if (['reonboard', 'associate'].includes(this.action)) {
-          return '';
-        }
+    return {trackAssociateOrManageAccountButton};
+  },
+  computed: {
+    hasAllShopsLinked() {
+      return this.unlinkedShops.length === 0;
+    },
+    hasAllShopsWithoutUrl() {
+      return this.shops.every((shop) => shop.domain === null);
+    },
+    hasShopsLinked() {
+      return this.hasShopsLinkedByUserInBackoffice || this.hasShopsLinkedWithoutEmployeeId;
+    },
+    hasShopsLinkedByUserInBackoffice() {
+      return this.shops.some(
+        (shop) => parseInt(shop.employeeId, 10) === this.backendUser.employeeId,
+      );
+    },
+    hasShopsLinkedWithoutEmployeeId() {
+      return this.shops.some(
+        (shop) => shop.uuid !== null && !shop.isLinkedV4 && shop.employeeId === null,
+      );
+    },
+    isLinkedV4() {
+      return this.shops.every((shop) => shop.isLinkedV4);
+    },
+    isShopContext() {
+      return this.shopContext === CONTEXT_SHOP;
+    },
+    specificUiUrl() {
+      if (['reonboard', 'associate'].includes(this.action)) {
+        return '';
+      }
 
-        if (['unlink'].includes(this.action)) {
-          return `/shop/${this.shops[0].uuid}`;
-        }
+      if (['unlink'].includes(this.action)) {
+        return `/shop/${this.shops[0].uuid}`;
+      }
 
-        return '/shop';
-      },
-      trackEventName() {
-        if (['reonboard', 'associate'].includes(this.action)) {
-          return '[ACC] Associate Button Clicked';
-        }
+      return '/shop';
+    },
+    trackEventName() {
+      if (['reonboard', 'associate'].includes(this.action)) {
+        return '[ACC] Associate Button Clicked';
+      }
 
-        if (['unlink'].includes(this.action)) {
-          return '[ACC] Unlink Shop Button Clicked';
-        }
+      if (['unlink'].includes(this.action)) {
+        return '[ACC] Unlink Shop Button Clicked';
+      }
 
-        return '[ACC] Manage Account Button Clicked';
-      },
-      unlinkedShops() {
-        return this.shops.filter((shop) => !shop.uuid || (shop.uuid && shop.isLinkedV4));
-      },
-      unlinkedShopsWithEmployeeId() {
-        return this.unlinkedShops.map((shop) => ({
-          ...shop,
-          employeeId: this.backendUser.employeeId.toString(),
-        }));
-      },
-    },
-    methods: {
-      openLinkShopModal() {
-        this.trackAssociateOrManageAccountButton();
+      return '[ACC] Manage Account Button Clicked';
+    },
+    unlinkedShops() {
+      return this.shops.filter((shop) => !shop.uuid || (shop.uuid && shop.isLinkedV4));
+    },
+    unlinkedShopsWithEmployeeId() {
+      return this.unlinkedShops.map((shop) => ({
+        ...shop,
+        employeeId: this.backendUser.employeeId.toString(),
+      }));
+    },
+  },
+  methods: {
+    openLinkShopModal() {
+      this.trackAssociateOrManageAccountButton();
 
-        this.cdcUiDisplayed = true;
-      },
```

Comment:
We have a regression here, we can't access the shops page from the "View my associated shops" button 
i think that during a merge we lost a part of the function code
You can fix that by adding these lines
```js
openLinkShopModal(action = 'associate') {
      this.action = action;
      this.trackAssociateOrManageAccountButton();

      this.cdcUiDisplayed = true;
    },
``